libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-detail-item/dh-processes-detail-item.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="child-row state-success">
+  <div class="child-cell">{{ detail.name }}</div>
+  <div class="child-cell">
+    <ng-container *ngIf="detail.createdDate">
+      <watt-icon
+        name="date"
+        [size]="iconSize.Small"
+        class="date-icon"
+      ></watt-icon>
+      <span class="watt-text-s">{{ detail.createdDate | dhDateTime }}</span>
+    </ng-container>
+  </div>
+  <div class="child-cell">
+    <ng-container *ngIf="detail.effectiveDate">
+      <watt-icon
+        name="date"
+        [size]="iconSize.Small"
+        class="date-icon"
+      ></watt-icon>
+      <span class="watt-text-s">{{ detail.effectiveDate | dhDate }}</span>
+    </ng-container>
+  </div>
+  <div class="child-cell details-status-cell">{{ detail.status }}</div>
+  <div class="child-cell open-logfile-cell">
+    <!-- TODO: Out comment and implement this when the "search log file" functionality is done -->
+    <!-- <watt-icon-->
+    <!--   name='openInNew'-->
+    <!--   [size]='iconSize.Medium'-->
+    <!--   class='open-in-new-icon'-->
+    <!-- ></watt-icon>-->
+  </div>
```

Comment:
I wanted to keep the column since it is part of "aligning" the columns in the table - And I figured someone would delete the column if there was no indication of why it was there.